core/services/relay/evm/tron_transmitter.go

```diff
@@ -0,0 +1,92 @@
+package evm
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	tronsdk "github.com/fbsobreira/gotron-sdk/pkg/address"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-evm/pkg/keys"
+	tronkeystore "github.com/smartcontractkit/chainlink-tron/relayer/keystore"
+	tron "github.com/smartcontractkit/chainlink-tron/relayer/ocr2"
+	tronclient "github.com/smartcontractkit/chainlink-tron/relayer/sdk"
+	trontxm "github.com/smartcontractkit/chainlink-tron/relayer/txm"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/legacyevm"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+type tronEVMTransmitterWrapper struct {
+	evmTransmitter ContractTransmitter
+}
+
+func (t *tronEVMTransmitterWrapper) LatestTransmissionDetails(ctx context.Context) (types.ConfigDigest, uint32, uint8, *big.Int, time.Time, error) {
+	configDigest, epoch, err := t.evmTransmitter.LatestConfigDigestAndEpoch(ctx)
+	if err != nil {
+		return types.ConfigDigest{}, 0, 0, nil, time.Time{}, err
+	}
+	return configDigest, epoch, 0, nil, time.Time{}, nil
+}
+
+func NewTronContractTransmitter(ctx context.Context, lggr logger.Logger, evmTransmitter ContractTransmitter, ethKeystore keys.Store, configWatcher *configWatcher, opts configTransmitterOpts, transmissionContractABI abi.ABI, ocrTransmitterOpts ...OCRTransmitterOption) (ContractTransmitter, error) {
```

Comment:
Let's polish the new code we add, it's going to be valuable in the future.

I'd extract this as `TronContractTransmitterOpts` type to document and make readable.